### PR TITLE
Adding has method to determine if object contains the current path.

### PR DESCRIPTION
**Pull request by woodburndesigns**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   },
   "homepage": "https://github.com/reflect/jason#readme",
   "license": "ISC",
-  "main": "index.js",
```
Comment (cp) [inline package.json:22]: I noticed this as well the other day... any clue how it could have worked before? 😕

File: src/index.js [hunk 2]
```diff
@@ -76,6 +76,19 @@ var _applyToPath = function(obj, path, val) {
...
+ * Queries an object and returns the true or false if the path is in the object
+ *
+ * @param  {Object} obj  An object to query.
+ * @param  {String} path A path to query against.
+ * @return {boolean}  A boolean reprsenting if the path is in the obj.
+ */
+var has = function(obj, path) {
+  var res = jp.query(obj, path);
+
+  return res.length > 0;
```
Comment (cp) [inline src/index.js:88]: It's pretty trivial, but do you think this is worth testing?

Comment (woodburndesigns) [reply src/index.js:88]: Definitely. Will add that right now.

**Review by cp — APPROVED** [review]

**Comment by cp** [comment]
👍

**Pull request merged**